scss/responsive/_small.scss

```diff
@@ -71,18 +77,35 @@
           }
         }
       }
+      .oo-text-color-selection-container {
+        min-width: 160px;
+        padding-left: $panel-padding;
+      }
+    }
+    .oo-caption-opacity-inner-wrapper {
+      top: 60%;
     }
     .oo-font-size-tab {
+      .oo-font-size-inner-wrapper {
+        top: calc(50% + 42px);
+      }
       .oo-selection-container-title {
         margin-bottom: 0px !important;
       }
     }
     .oo-text-enhancements-tab {
+      .oo-text-enhancements-inner-wrapper {
+        top: calc(50% + 42px);
+      }
       .oo-selection-container-title {
         margin-bottom: 0px !important;
       }
     }
   }
+  //CC Preview Panel
+  .oo-preview-panel {
+    display: none;
```

Comment:
Correct, we are removing the cc-preview on small and extra small devices because there is just not enough room.